runtime/compiler/env/j9method.cpp

```diff
@@ -6557,10 +6557,8 @@ TR_ResolvedJ9Method::getResolvedSpecialMethod(TR::Compilation * comp, I_32 cpInd
    // See if the constant pool entry is already resolved or not
    //
    J9Method * ramMethod;
-   if (unresolvedInCP)
+   if (unresolvedInCP != NULL)
       {
-      ramMethod = jitGetJ9MethodUsingIndex(_fe->vmThread(), cp(), cpIndex);
```

Comment:
This code seems questionable both before and after my change in that it never looks at the method it looked up and always sets `unresolvedInCP` to true.

Based on the code below the intent might have been to default to unresolved?  In that case the rest of the comment can be removed, I think.

Looking for opinions from the reviewers on the right change here

Reply:
I agree, looking at the code it seems to be defensive in that we default to unresolved unless we prove otherwise. The comment does not really make sense to me though.

Reply:
Thanks for the confirmation @fjeremic.  I've pushed a new commit that removes that comment entirely